src/test/java/nl/hsac/fitnesse/util/iban/ATIbanTest.java

```diff
@@ -0,0 +1,65 @@
+package nl.hsac.fitnesse.util.iban;
+
+import nl.hsac.fitnesse.util.RandomUtil;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests AT Iban generator.
+ */
+public class ATIbanTest {
+    private final ATIban generator = new ATIban();
+    private static final RandomUtil RANDOM_UTIL = new RandomUtil();
+
+    @Rule
+    public final ExpectedException exception = ExpectedException.none();
+
+    /**
+     * Tests generation without parameters.
+     */
+    @Test
+    public void testNoParam() {
+        for (int i = 0; i < 100; i++) {
+            String result = generator.generateATIban("", "");
+            assertEquals("Got: " + result, 20, result.length());
+            assertTrue("Got: " + result, result.charAt(0)=='A' && result.charAt(1)=='T');
+        }
+    }
+
+    /**
+     * Tests generation without bankCode.
+     */
+    @Test
+    public void testNoBankCode() {
+        for (int i = 0; i < 100; i++) {
+            String result = generator.generateATIban("AT", "");
+            assertEquals("Got: " + result, 20, result.length());
+        }
+    }
+
+    /**
+     * Tests basic generation.
+     */
+    @Test
+    public void testGenerate() {
+        for (int i = 0; i < 100; i++) {
+            String bic = RANDOM_UTIL.randomElement(generator.ATBankCodes);
+            String result = generator.generateATIban("AT", bic);
+            assertEquals("Got: " + result, 20, result.length());
```

Comment:
Do you want to check for country prefix and presence of selected `bic`?
By the way I believe the 'BIC' is not the same as the bank code in letters (see for instance https://www.betaalvereniging.nl/giraal-en-online-betalen/sepa-documentatie-voor-nederland/bic-afleiden-uit-iban/), so this name is maybe a bit confusing...